pandas/core/frame.py

```diff
@@ -5015,7 +5015,7 @@ def rename(
     def fillna(
         self,
         value=...,
-        method: str | None = ...,
+        method: Literal["backfill", "bfill", "ffill", "pad"] | None = ...,
```

Comment:
is it possible to make this a type alias to avoid all of this repeating? (i mean the literal itself)

Reply:
I've done that.  It's a bit awkward because we have to treat `Literal` in a special way to preserve python 3.7 compatibility.  Addressed in commit https://github.com/pandas-dev/pandas/pull/41255/commits/c216bd911417a127a00a3e069850a1e4d9855a9e

Reply:
got